src/modules/configuration/index.ts

```diff
@@ -139,6 +140,14 @@ export const configValues = () => ({
           acrossVersion: "2.5",
         },
       ],
+      [ChainIds.zkSyncTestnet]: [
+        {
+          address: "0x863859ef502F0Ee9676626ED5B418037252eFeb2",
+          startBlockNumber: 5000000,
+          abi: JSON.stringify(GoerliSpokePool2_5Abi),
+          acrossVersion: "2.5",
+        },
+      ],
```

Comment:
I think we can already add support for zk sync mainnet. 
```
0xE0B015E54d54fc84a6cB9B666099c46adE9335FF
```

See https://github.com/across-protocol/contracts-v2/pull/338/files

Reply:
Done [here](https://github.com/across-protocol/scraper-api/pull/246/commits/9b4b06645986409c9ed518c94637f3800d5b8706)